server/routes/appointments/video-link-booking/handlers/confirmation.ts

```diff
@@ -10,7 +10,10 @@ export default class ConfirmationRoutes {
     req.session.bookAVideoLinkJourney = null
 
     const vlb = await this.bookAVideoLinkService.getVideoLinkBookingById(+vlbId, user)
+    const court = await this.bookAVideoLinkService
```

Comment:
We don't have an endpoint to get a single court.  I could create one but this is the only place it is needed ...